cfgov/v1/models/filterable_list_mixins.py

```diff
@@ -95,7 +97,10 @@ def get_context(self, request, *args, **kwargs):
 
         form_data, has_active_filters = self.get_form_data(request.GET)
         queryset = self.get_filterable_queryset()
-        has_archived_posts = queryset.filter(is_archived='yes').count() > 0
+        if flag_enabled('ARCHIVE_FILTER_OPTIONS'):
```

Comment:
Can we add a comment above here that this flag check is so that we can enable or disable the the archive filter options?

Also, if you pass `flag_enabled` the request, it'll possible to condition it on request-dependent properties in the future.
```suggestion
        if flag_enabled('ARCHIVE_FILTER_OPTIONS', request=request):
```

Reply:
Added comment and request to flag check